### DISCUSSION TITLE
Website favicon fix

### DISCUSSION THREAD

**Discussion by Camell-Cloud** [Camell-Cloud/Camell-web3#10] [Q&A] [answered]
Website's favicon is still the default icon could anyone change it?

<img width="98" alt="image" src="https://github.com/user-attachments/assets/ed5d240c-e3ee-4764-96c2-a120bb9273de" />

**Comment by jeonilshin** [comment, marked as answer]
done!

![image](https://github.com/user-attachments/assets/c8b2fabf-f852-4ae5-a11b-c792b8de6bfd)

**Reply by Camell-Cloud** [reply]
Thankyou very much!!
